components/06-pages/03-application-form/application-preferences--custom.html

```diff
@@ -21,14 +21,18 @@ <h2 class="app-card_question">Your household may qualify for the following lotte
             <div class="checkbox-group">
               <div class="">
                 <div class="checkbox-block margin-bottom--2x">
-                  <input aria-decribedby="description-nhrp" type="checkbox" name="preferences-nhrp" value="preferences-nhrp" id="preferences-nhrp">
+                  <input aria-decribedby="description-custom" type="checkbox" name="preferences-custom" value="preferences-custom" id="preferences-custom">
                   <label class="checkbox-block_label" for="preferences-displaced">
                     <strong class="form-label caps">Some Custom Preference</strong>
                   </label>
                   <div class="checkbox-block_note-inner">
-                    {{> @more classes="open-bottom margin-top-none" icon="arrow-down" size="tiny" fill="primary"}}
-                    <p class="checkbox-block_more toggled" id="description-nhrp" style="display:none;">
-                      At least one household member must currently work in San Francisco at least 75% of their working hours.
+                    <p class="checkbox-block_more" id="description-custom">
```

Comment:
what ended up being the solution to the issue you were having?

Reply:
I removed the "read more" and changed the css to target any `ul` in checkbox-note instead of restricting it to `checkbox-block_more`